### DISCUSSION TITLE
How to integrate react-select isMulti?

### DISCUSSION THREAD

**Discussion by ttizze** [edmundhung/conform#764] [Q&A] [answered]
Hi!
I want to use conform with react-select　isMulti.
but i can't.
Please Help!

**Comment by shafkathullah** [comment]
i think this lib is for native html form elements only, as i have not seen any mention of custom component support.

**Reply by Pouet--** [reply]
I use conform functions like getInputProps() on custom components, either from radix or react-aria. 
But your component has to be a HTML element and it depends how it is configured.

Maybe you can learn from [this discussion](https://github.com/edmundhung/conform/discussions/411) or looking for custom component integration

**Comment by Pouet--** [comment, marked as answer]
The conform guide provides help for integrating UI libraries Components : [UI-integration](https://conform.guide/integration/ui-libraries)

**Reply by shafkathullah** [reply]
oh really missed that while checking doc.
